src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid"
+import "./App.css";
+import Navbar from "./Navbar";
+import Bttn from "./Button";
+import Homepage from "./Homepage";
+import Onecard from "./Card";
+import cards from "./cards.json";
+
+function App() {
+  return (
+    <div className="App">
+      <div className="container">
+        <div >
+          <Navbar />
+        </div>
+        <div>
+          <Homepage />
+        </div>
+        <div>
+          <Bttn />
+        </div>
+      </div>
```

Comment:
You don't need to wrap everything in another <div>, you're already doing that in the components.